src/kartikeyDJune20/FirstDifferIndexArray.java

```diff
@@ -0,0 +1,48 @@
+package kartikeyDJune20;
+
+public class FirstDifferIndexArray {
+
+	void firstDifferIndex(int[] input1, int[] input2) {
+
+		if (input1.length == input2.length) {
+
+			boolean flag = true;
```

Comment:
Flag variable is not needed, please remove it.

Reply:
i have used the flag to Print .. if in case there is no difference and both the arrays are identical.. Rest of the sugggested changes are incorporated.